sdk/build.gradle

```diff
@@ -90,13 +90,14 @@ subprojects {
         }
 
         def fileFilter = ['**/R.class', '**/R$*.class', '**/BuildConfig.*', '**/Manifest*.*', '**/*Test*.*', 'android/**/*.*']
-        sourceDirectories = files(["$projectDir/src/main/java"])
-        classDirectories = files([
+        sourceDirectories.from = files(["$projectDir/src/main/java"])
+        classDirectories.from = files([
                 fileTree(dir: "$buildDir/intermediates/classes/debug", excludes: fileFilter),
                 fileTree(dir: "$buildDir/intermediates/javac/debug/compileDebugJavaWithJavac/classes", excludes: fileFilter)
         ])
-        executionData = fileTree(dir: buildDir, includes: [
+        executionData.from = fileTree(dir: buildDir, includes: [
                 'jacoco/testDebugUnitTest.exec',
+                'outputs/code_coverage/debugAndroidTest/connected/*coverage.ec',
                 'outputs/code-coverage/connected/*coverage.ec'
```

Comment:
We keep the old one for previous android versions or is there another reason?

Reply:
Let's add a comment then.

Reply:
Hum can you test with only the new path? Since we use the latest android plugin it should not have to handle previous versions.